software/contrib/pong.py

```diff
@@ -0,0 +1,230 @@
+"""
+Pong
+author: Igor Nawrocki (github.com/Bugari)
+date: 2023-05-19
+labels: game
+
+EuroPi version of the classic Pong game.
+
+digital_in: unused
+analog_in: unused
+
+knob_1: controls position of the left paddle
+knob_2: controls position of the right paddle
+
+button_1: start a new game
+button_2: unused
+
+output_1: trigger a sound for the ball hitting a wall
+output_2: trigger a sound for the ball hitting a paddle
+output_3: trigger a sound for scoring
+output_4: unused
+output_5: unused
+output_6: unused
+
+"""
+
+import math
+import time
+import random
+from time import sleep
+from europi import oled, b1, k1, k2, Output, cv1, cv2, cv3
+from europi_script import EuroPiScript
+
+
+class Paddle:
```

Comment:
[Optional] I'm not sure that this gets any benefit from being a class, I think that it would work just as well as three 'constants'.